staging/src/k8s.io/client-go/tools/clientcmd/config.go

```diff
@@ -379,6 +379,17 @@ func (p *persister) Persist(config map[string]string) error {
 	return nil
 }
 
+func NoopPersister() restclient.AuthProviderConfigPersister {
```

Comment:
if we move use to `GetAuthProvider `, I would also probably keep the no-op persisted implementation private to k8s.io/client-go/rest

Reply:
done